fv_be/app/admin/characters_admin.py

```diff
@@ -46,6 +48,7 @@ class CharacterInline(BaseInlineAdmin):
         "sort_order",
         "approximate_form",
         "notes",
+        "related_dictionary_entries",
     ) + BaseInlineAdmin.fields
     readonly_fields = BaseInlineAdmin.readonly_fields + CharacterAdmin.readonly_fields
 
```

Comment:
Just a suggestion but the default admin display for ManyToMany fields is somewhat unintuitive and it may be useful to add `filter_horizontal = ("related_dictionary_entries",)` here. It might also be worth adding this to the CharacterInline as well.

Reply:
I just added that it's a much better UI. Thanks!